### DISCUSSION TITLE
Is it possible to retrieve "Recorded On" info from "Attendance Reports" as a shortcode?

### DISCUSSION THREAD

**Discussion by refugeindhamma** [liedekef/events-made-easy#276] [Q&A] [answered]
Is it possible to retrieve "Recorded On" info from "Attendance Reports" as a shortcode?

I would like to add this info to emails to attendees who are late.

**Comment by liedekef** [comment, marked as answer]
Currently the attendance reporting is not accessible via a shortcode. Feel free to support me to extend this functionality :-)
I also think that it can be made accessible via the backend for events, but currently also not the case unfortunately.

**Reply by refugeindhamma** [reply]
Thank you for the reply.
